cypress/integration/getTokenSilently.js

```diff
@@ -82,7 +82,8 @@ describe('getTokenSilently', function() {
             cy.toggleSwitch('use-cache');
 
             cy.login().then(() => {
-              cy.toggleSwitch('refresh-tokens').wait(250);
+              cy.toggleSwitch('refresh-tokens').wait(1000);
```

Comment:
is this also related to the 5 sec delay comment above?

Reply:
Yes, seems to have cured an intermittant failure. At least locally..